var/spack/repos/builtin/packages/thrust/package.py

```diff
@@ -11,8 +11,22 @@ class Thrust(Package):
     which resembles the C++ Standard Template Library (STL)."""
 
     homepage = "https://thrust.github.io"
-    url      = "https://github.com/thrust/thrust/archive/1.9.4.tar.gz"
+    url      = "https://github.com/NVIDIA/thrust/archive/1.12.0.tar.gz"
 
+    version('1.12.0', sha256='d68f89937ed1a0dadafd4f77d3e65ab9a5baa48dd5fa5698f8d3f7e3d6749da2')
+    version('1.12.0-rc0', sha256='458c5ccc8572af00b3176af0f45f5f7eeb5af737bcc0a853982b55afcbe4cef6')
```

Comment:
Can we leave out `rc` and `-1` tags? Or are they needed for any specific reason?

Reply:
Thanks for your advice.:grinning: 

For `rc` tags, I'll just remove them. But I'm not sure about the `-1` tags, e.g. `1.9.8-1` is a variant of 1.9.8 with some necessary modifications to accompany the NVIDIA HPC SDK. So could we just retain these `-1` tags?

Some link for reference:
https://github.com/NVIDIA/thrust/releases/tag/1.9.8-1
https://github.com/NVIDIA/thrust/releases/tag/1.9.6-1